string name
string url
float64[] pose